CONTRIBUTING.md

```diff
@@ -1,37 +1,57 @@
-## Adding a new API
```

Comment:
@JustinBeckwith @kinwa91 this had some custom help for this repo. Should we skip this PR, or somehow append the new stuff without erasing the old?

Reply:
Yeah - would you mind adding this to the ignored files in the synth file?

Reply:
Sure. How?

Reply:
Pretty much like this: https://github.com/googleapis/google-api-nodejs-client/blob/master/synth.py#L27

Reply:
Thanks! https://github.com/googleapis/nodejs-proto-files/pull/154